tooling/prettier/index.mjs

```diff
@@ -3,7 +3,7 @@
 
 /** @type { PrettierConfig | SortImportsConfig } */
 const config = {
-  plugins: ["@ianvs/prettier-plugin-sort-imports"],
+  plugins: ["@ianvs/prettier-plugin-sort-imports", "prettier-plugin-packagejson"],
```

Comment:
Okay now I understand why it's ordering the things in the package.json, can you put this in a seperate pr?

Reply:
Yes, will put in a seperate